snapd-glib/snapd-client-sync.c

```diff
@@ -519,6 +519,36 @@ snapd_client_get_interfaces_sync (SnapdClient *client,
     return snapd_client_get_interfaces_finish (client, data.result, plugs, slots, error);
 }
 
+/**
+ * snapd_client_get_interface_info_sync:
+ * @client: a #SnapdClient.
+ * @names: (array zero-terminated=1): a null-terminated array of interface names or %NULL.
```

Comment:
Missing (allow-none). Also if NULL is provided what happens? Should it say ".. or %NULL for all interfaces"?